stac_fastapi/extensions/stac_fastapi/extensions/core/transaction.py

```diff
@@ -16,6 +17,17 @@
 from stac_fastapi.types.extension import ApiExtension
 
 
+class TransactionConformanceClasses(str, Enum):
+    """Conformance classes for the Transaction extension.
+
+    See https://github.com/stac-api-extensions/transaction
+
+    """
+
+    ITEMS = "https://api.stacspec.org/v1.0.0/ogcapi-features/extensions/transaction"
+    COLLECTIONS = "https://api.stacspec.org/v1.0.0/collections/extensions/transaction"
```

Comment:
Note: in `pgstac` we can `register` **search** request. This is used in titiler-pgstac where we register search request to get a uniq ID then used to get map tiles. It might be interesting to work on a transaction specification for `search` 🤷 

cc @bitner @hrodmn